src/chains/filecoin/filecoin/package.json

```diff
@@ -28,6 +29,7 @@
     "directory": "src/chains/filecoin"
   },
   "scripts": {
+    "build": "npx --node-arg=--max-old-space-size=10000 webpack",
```

Comment:
I don't think we need `npx`, if I'm right, then something like:

```json
    "build": "cross-env NODE_OPTIONS=--max-old-space-size=10000 webpack",
```

should do the trick.

Also, 10000 is quite a lot. haha

Reply:
Ah, yes that should do the trick

I got frustrated after bumping it twice and just moved it to ~10gib and never look backed lol. I might be able to make this smaller now that all of the `ipfs` deps were externalized; they likely were the culprits of making this larger.